src/iroh.rs

```diff
@@ -380,6 +474,161 @@ impl VeilidIrohBlobs {
 
         return Ok(read);
     }
+    pub async fn create_collection(&self, collection_name: String) -> anyhow::Result<Hash> {
+        
+        println!("Creating a new empty collection with name: {}", collection_name);
+        
+        // Create a new empty collection
+        let collection = Collection::default();
```

Comment:
Instead of Collection we should use HashMap and save/load that hash map as CBOR like it was a regular file